test/hotspot/jtreg/runtime/whitebox/TestWBDeflateIdleMonitors.java

```diff
@@ -59,14 +59,22 @@ public static void main(String args[]) throws Exception {
 
     public static class InflateMonitorsTest {
         static WhiteBox wb = WhiteBox.getWhiteBox();
+        static long LockingMode = wb.getIntVMFlag("LockingMode");
+        static long LM_LIGHTWEIGHT = 2;
         public static Object obj;
 
-        public static void main(String args[]) {
+        public static void main(String args[]) throws Exception {
             obj = new Object();
             synchronized (obj) {
-                // HotSpot implementation detail: asking for the hash code
-                // when the object is locked causes monitor inflation.
-                if (obj.hashCode() == 0xBAD) System.out.println("!");
+                if (LockingMode != LM_LIGHTWEIGHT) {
```

Comment:
I don't think we need to make this test behave differently because of implementation details of different locking modes. Just use `wait(1)` unconditionally. Thanks.

Reply:
Done.